src/pipx/util.py

```diff
@@ -112,19 +112,26 @@ def run_pypackage_bin(bin_path: Path, args: List[str]) -> NoReturn:
     )
 
 
-if WINDOWS:
-
-    def get_venv_paths(root: Path) -> Tuple[Path, Path]:
-        bin_path = root / "Scripts"
-        python_path = bin_path / "python.exe"
-        return bin_path, python_path
-
-else:
-
-    def get_venv_paths(root: Path) -> Tuple[Path, Path]:
-        bin_path = root / "bin"
-        python_path = bin_path / "python"
-        return bin_path, python_path
+def get_venv_paths(python: str, root: Path) -> Tuple[Path, Path]:
+    command_str = textwrap.dedent(
+        f"""
+        import venv
+        root_paths = venv.EnvBuilder().ensure_directories(r"{str(root)}")
```

Comment:
```suggestion
        root_paths = venv.EnvBuilder().ensure_directories({root})
```

Reply:
Probably raw string should be used? (For handling paths on Windows)